st2actions/st2actions/query/mistral/v2.py

```diff
@@ -74,7 +74,7 @@ def query(self, execution_id, query_context):
             raise
 
         try:
-            tasks = self._get_workflow_tasks(mistral_exec_id)
+            result['extra']['tasks'] = self._get_workflow_tasks(mistral_exec_id)
```

Comment:
Can we do this part of a separate PR / later?

Since this is backward incompatible and I believe it will also affect the current CLI output...